runtime/common/src/lib.rs

```diff
@@ -260,3 +262,9 @@ parameter_types! {
 	pub const MinTotalFarmRewards: Balance = NATIVE_EXISTENTIAL_DEPOSIT * 1_000;
 	pub const NftClass: primitives::ClassId = 1;
 }
+
+// Stableswap
+parameter_types! {
+	pub const StableswapPrecision: Balance = 1u128;
+	pub const StableswapAmplificationRange: RangeInclusive<u32> = RangeInclusive::new(2,10000);
```

Comment:
Do you want to pick `u32` for the amplicfication? Seems like you could easily fit `10_000` into a `u16`.
```suggestion
	pub const StableswapAmplificationRange: RangeInclusive<u32> = RangeInclusive::new(2,10_000);
```

Reply:
yes, possibly. we had a discussion what could be max amplification value. 

u16::MAX should be sufficient regardless. I'll change.